src/include/sof/audio/component.h

```diff
@@ -223,70 +222,159 @@ enum comp_copy_type {
 struct comp_driver;
 
 /**
- * Audio component operations - all mandatory.
+ * Audio component operations.
  *
  * All component operations must return 0 for success, negative values for
- * errors and 1 to stop the pipeline walk operation.
+ * errors and 1 to stop the pipeline walk operation unless specified otherwise
+ * in the operation documentation.
```

Comment:
This particular fix in the documentation may warrant a separate commit.

Reply:
ok